cmd/serve.go

```diff
@@ -23,6 +23,7 @@ func init() {
 func initServerFlags(cmd *cobra.Command) {
 	cmd.Flags().IntP("port", "p", DEFAULT_PORT, "The port for sidekick to listen on.")
 	cmd.Flags().BoolP("local", "l", false, "Run sidekick in local (non cloud) mode. This is mostly use for testing locally.")
+	cmd.Flags().StringP("localipaddress", "i", "", "Use local ipaddr to get the local bolt address.")
```

Comment:
Consider giving this a more descriptive name? it is a address of the local data cruncher instance right? and is used to override the Bolt endpoints so perhaps something like `bolt-endpoint-override` or `bolt-endpoint`? Ditto this in the help message - we're forcing the ip address we pass here to be the Bolt address, I don't think that's super clear at the moment.

Also consider adding dashes vs concatenating 3 words for ux & readability - https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html

Given this is public & cx-facing I think we should be mindful of these details.

Reply:
I agree, let's go with bolt-endpoint-override.